src/containerapp/azext_containerapp/_params.py

```diff
@@ -44,6 +44,11 @@ def load_arguments(self, _):
         c.argument('service_principal_client_secret', help='The service principal client secret. Used by GitHub Actions to authenticate with Azure.', options_list=["--service-principal-client-secret", "--sp-sec"])
         c.argument('service_principal_tenant_id', help='The service principal tenant ID. Used by GitHub Actions to authenticate with Azure.', options_list=["--service-principal-tenant-id", "--sp-tid"])
 
+    # Runtime
+    with self.argument_context('containerapp create', arg_group='Runtime', is_preview=True) as c:
+        c.argument('runtime', help='The runtime of the container app. Supported values are "generic" and "java".', is_preview=True)
```

Comment:
If the param is enumerable, please use `arg_type=get_enum_type()` to define it. It will prompt users for the available values and automatically validate if the input meets expectation.
```suggestion
        c.argument('runtime', arg_type=get_enum_type(['generic', 'java']), help='The runtime of the container app.', is_preview=True)
```

Reply:
Hi @felixsong0824 

Please take a look for these comments.

Reply:
@yanzhudd comments fixed, please help review again.